tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "experimentalDecorators": true,
     "importHelpers": true,
     "inlineSourceMap": true,
-    "lib": ["dom", "es2020"],
+    "lib": ["dom", "es2020", "dom.iterable"],
```

Comment:
Note: this was required to support for doing `for ... of` over a FormData in Typescript